charts/keptn-lifecycle-toolkit/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
shouldn't this file come from the push from keptn/lifecycle-toolkit?

Reply:
yes, but then we need .gitkeep file here

Reply:
exactly

Reply:
ok